cmd/main.go

```diff
@@ -0,0 +1,93 @@
+package cmd
+
+import (
+	"context"
+	"encoding/hex"
+	"fmt"
+	"os"
+	"time"
+
+	client "github.com/astriaorg/go-sequencer-client/client"
+)
+
+func Execute() {
+	if len(os.Args) < 2 {
+		fmt.Println("Expected a command.")
+		printHelp()
+		os.Exit(1)
+	}
+
+	switch os.Args[1] {
+
+	case "createaccount":
+		handleCreateAccount()
+	case "getbalance":
+		handleGetBalance()
+	default:
+		printHelp()
+		os.Exit(1)
+	}
+
+}
+
+func printHelp() {
+	fmt.Println("Usage: go-sequencer-client-cli <command>")
+	fmt.Println("Available commands are:")
+	fmt.Println("  createaccount                       : creates an account")
+	fmt.Println("  getbalance <rpc_endpoint> <address> : gets the balance of an account")
+	os.Exit(1)
+}
+
+func handleCreateAccount() {
+	privateKey, err := client.NewAccount()
```

Comment:
so then here you can just do `client.GenerateSigner()` then `signer.PrivateKey()`

Reply:
Should I get rid of `NewSigner` then too?

Reply:
it's fine to leave, there might be other usages for it